### ISSUE TITLE
Remove In[]/Out[] from Markdown

### ISSUE THREAD

**Issue by maxitg** [maxitg/SetReplace#665]
### The problem

Since Github now supports click-to-copy for code blocks, it became somewhat inconvenient that the input/output code blocks contain `In[] :=` and `Out[] =` prefixes that should not be copied.

It might be better to remove them altogether. It will make it less clear where the inputs/outputs are, but it needs to be tested to see whether it's more important than click-to-copy ability.

Also, it might require regenerating pictures without the labels for consistency.

### Alternative solutions

It would be ideal to have it copy the code without the prefixes while displaying prefixes nonetheless. But that's likely too difficult or not possible.

**Comment by daneelsan** [comment]
So ideally it should work like the WFR documentation? I.e. only allow copy for the input

![image](https://user-images.githubusercontent.com/40190339/126815614-50b1fe34-3bc7-4a2c-8cab-ffad9d57ea93.png)

**Comment by maxitg** [comment]
@daneelsan, yes, but it [won't be possible](https://stackoverflow.com/questions/21340803/embed-javascript-in-github-readme-md) to do it on GitHub directly.

**Comment by daneelsan** [comment]
I meant generating this:
```wl
(*In[] :=*) test[]
```
```wl
(*Out[] =*) "This is a test"
```
It would be great to modify what is/isn't copied....

**Comment by maxitg** [comment]
@daneelsan, yes, I think that's better than omitting them entirely. It will also allow us to keep images the same.
